src/csvwidget/widget.ts

```diff
@@ -137,15 +141,21 @@ class CSVWidget extends Widget {
       th.textContent = name;
       header.appendChild(th);
     }
-    for (let row of parsed) {
+    for (let row of parsed.slice(0, RENDER_LIMIT)) {
       let tr = document.createElement('tr');
       for (let col of parsed.columns) {
         let td = document.createElement('td');
-        td.textContent = row[col]
+        td.textContent = row[col];
         tr.appendChild(td);
       }
       body.appendChild(tr);
     }
+    if (parsed.length > RENDER_LIMIT) {
```

Comment:
How does this look?

<img width="454" alt="screen shot 2016-09-28 at 7 43 26 am" src="https://cloud.githubusercontent.com/assets/2096628/18913768/51a3edf6-854f-11e6-82cf-31d35980bcd5.png">